cf-stacks/compute-stack.yml

```diff
@@ -61,6 +61,8 @@ Resources:
               Value: !Sub '{{resolve:ssm:/hei-admin-api/${Env}/db/url}}'
             - Name: SPRING_DATASOURCE_PASSWORD
               Value: !Sub '{{resolve:ssm:/hei-admin-api/${Env}/db/password}}'
+            - Name: AWS_SES_SOURCE
+              Value: !Sub '{{resolve:ssm:/hei-admin-api/${Env}/ses/source}}'
```

Comment:
In the future, put the changes impacting CI and CD file configuration into specific commits.